pfs/camera/camera.yaml

```diff
@@ -209,3 +209,54 @@ CCDs : &CCDs
         <<: *CCD
         id : 2
         offset : [-93.1, 65.1]           # mm
+
+    b2:
+      <<: *CCD
+      id: 4
+      offset : [-28.0, -65.1]         # mm
+    r2:
+      <<: *CCD
+      id: 5
+      offset : [-28.0, 0.0]           # mm
+    n2:
+      <<: *CCD
+      id: 6
+      offset : [-28.0, 65.1]           # mm
+    m2:
+      <<: *CCD
+      id: 7
+      offset : [-28.0, 130.2]         # mm
+
```

Comment:
Looks like you are following a numbering scheme for the`id`  such that it increases in order of `b, r, n m` for SMs 2, 3 and 4. However for SM1 the order is `r, b, n, m`. That is, `id=0` corresponds to `r1`, `id=1` is `b1`, etc.

Shouldn't the ids follow a consistent numbering scheme?